psalm.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<psalm
```

Comment:
I guess pipeline itself should be adjusted: https://github.com/yiisoft/cache-memcached/blob/master/.github/workflows/static.yml